internal/support/scheduler/utils.go

```diff
@@ -48,6 +49,30 @@ func msToTime(ms string) (time.Time, error) {
 	return time.Unix(0, msInt*int64(time.Millisecond)), nil
 }
 
+// Frequency indicates how often the specific resource needs to be polled.
+// It represents as a duration string.
+//
+// Nanosecond Duration = 1
+// Microsecond = 1000 * Nanosecond
+// Millisecond = 1000 * Microsecond
+// Second = 1000 * Millisecond
+// Minute = 60 * Second
+// Hour = 60 * Minute
+func parseFrequency(durationStr string) (time.Duration, error) {
+
+	// default
+	var durationDefault = 24 * time.Hour
+
+	// check Frequency
+	timeDuration, err := time.ParseDuration(durationStr)
+	if err != nil {
+		LoggingClient.Error("parse frequency error, the original time string is : " + durationStr)
```

Comment:
This is the same output that your error handling code is going to have. Your code will output the same error alert twice, both without any real insight into what exactly is unparsable.

Reply:
fixed